crates/types/src/rpc/transactions/broadcasted_deploy_account_transaction_v3.rs

```diff
@@ -0,0 +1,196 @@
+use std::sync::Arc;
+
+use serde::{Deserialize, Serialize};
+use starknet_api::core::calculate_contract_address;
+use starknet_api::transaction::DeployAccountTransactionV3;
+use starknet_rs_crypto::poseidon_hash_many;
+
+use super::broadcasted_deploy_account_transaction_v1::PREFIX_DEPLOY_ACCOUNT;
+use super::BroadcastedTransactionCommonV3;
+use crate::contract_address::ContractAddress;
+use crate::error::DevnetResult;
+use crate::felt::{Calldata, ClassHash, ContractAddressSalt, Felt};
+use crate::utils::into_vec;
+
+#[derive(Debug, Clone, Eq, PartialEq, Deserialize, Serialize)]
```

Comment:
For this and other similar structs, do we need all these derivations? E.g. `Serialize`? Related: https://github.com/0xSpaceShard/starknet-devnet-rs/issues/188

Reply:
For Serialize its needed, because of dump feature

Reply:
I think we don't dump broadcasted txs. I just tried removing this derivation (and a few others which dependend on it), and `cargo check` didn't complain about it.

Maybe this can be addressed in the future, with other things noted in #188.

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/2e7f1455f21b277b287f12dfa6c4c7133031ac8b

Reply:
Cool - I see you applied this to various places. Should #188 be marked as closeable or should someone do a more thorough check of the repo as a separate contribution?

Reply:
this doesn't remove unnecessary derivations entirely. The issue shouldn't be marked as closable.